### DISCUSSION TITLE
is there a way to set the variable default select "ALL"

### DISCUSSION THREAD

**Discussion by keJ2011** [grafana/grafana#69531] [Dashboards, Panels & Visualizations]
I have two variable, the first default select ALL, but the second is not. like this:
<img width="559" alt="image" src="https://github.com/grafana/grafana/assets/6027970/853c7352-6f0e-4972-8071-28a8c7768678">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
